util/pkgconfig/seqan.pc.in

```diff
@@ -6,5 +6,6 @@ Name: @CMAKE_PROJECT_NAME@
 Description: C++ library for biological sequence analysis
 URL: http://www.seqan.de
 Version: @SEQAN_VERSION_STRING@
-Requires: zlib
-Cflags: -I${includedir}
+Requires: @SEQAN_PKGCFG_DEPS@
+Cflags: -I${includedir}  @SEQAN_PKGCFG_CFLAGS@ -DSEQAN_HAS_ZLIB=1 -DSEQAN_HAS_BZIP2=1 -DSEQAN_HAS_OPENMP=1
```

Comment:
Cool, I think this is ready to roll now. Would be worth double checking by hand that it all work at your end (on your build boxes), as I've only checked this on a couple of systems, with recent pkg-config and cmake versions.